tests/storage/reservation.go

```diff
@@ -0,0 +1,295 @@
+	}
+	BeforeEach(func() {
+		fgDisabled = !checks.HasFeature(virtconfig.PersistentReservation)
+		if fgDisabled {
+			tests.EnableFeatureGate(virtconfig.PersistentReservation)
+		}
+
+	})
+	AfterEach(func() {
+		if fgDisabled {
+			tests.DisableFeatureGate(virtconfig.PersistentReservation)
+			testsuite.WaitVirtHandlerReady()
+		}
+	})
+
+	Context("Use LUN disk with presistent reservation", func() {
+		BeforeEach(func() {
+			var err error
+			virtClient, err = kubecli.GetKubevirtClient()
+			Expect(err).ToNot(HaveOccurred())
+			naa = generateNaa()
+			backendDisk = "disk" + rand.String(randLen)
+			disk = "disk-" + rand.String(randLen)
+			targetCliPod = "targetcli-" + rand.String(randLen)
+			// Create the scsi disk
+			createSCSIDisk(targetCliPod, disk)
+			// Avoid races if there is some delay in the device creation
+			Eventually(findSCSIdisk(targetCliPod, backendDisk), 20*time.Second, 1*time.Second).ShouldNot(BeEmpty())
+			device = findSCSIdisk(targetCliPod, backendDisk)
+			Expect(device).ToNot(BeEmpty())
+			By(fmt.Sprintf("Create PVC with SCSI disk %s", device))
+			pv, pvc, err = tests.CreatePVandPVCwithSCSIDisk(node, device, util.NamespaceTestDefault, "scsi-disks", "scsipv", "scsipvc")
+			Expect(err).ToNot(HaveOccurred())
+			waitForVirtHandlerWithPrHelperReadyOnNode(node)
+		})
+
+		AfterEach(func() {
+			// Delete the scsi disk
+			executeTargetCli(targetCliPod, []string{
+				"loopback/", "delete", naa})
+			executeTargetCli(targetCliPod, []string{
+				"backstores/fileio", "delete", backendDisk})
+			Expect(virtClient.CoreV1().PersistentVolumes().Delete(context.Background(), pv.Name, metav1.DeleteOptions{})).NotTo(HaveOccurred())
+
+		})
+
+		It("Should successfully start a VM with persistent reservation", func() {
+			By("Create VMI with the SCSI disk")
+			vmi := libvmi.NewFedora(
+				libvmi.WithPersistentVolumeClaimLun("lun0", pvc.Name, true),
+			)
+			vmi.Namespace = util.NamespaceTestDefault
+			vmi = tests.CreateVmiOnNode(vmi, node)
+			libwait.WaitForSuccessfulVMIStartWithTimeoutIgnoreWarnings(vmi, 180)
+			By("Requesting SCSI persistent reservation")
+			Expect(console.LoginToFedora(vmi)).To(Succeed(), "Should be able to login to the Fedora VM")
+			Expect(checkResultCommand(vmi, "sg_persist -i -k /dev/sda",
+				"there are NO registered reservation keys")).To(BeTrue())
+			Expect(checkResultCommand(vmi, "sg_persist -o -G  --param-sark=12345678 /dev/sda",
+				"Peripheral device type: disk")).To(BeTrue())
+			Eventually(checkResultCommand(vmi, "sg_persist -i -k /dev/sda",
+				"1 registered reservation key follows:\r\n    0x12345678\r\n"),
+				60*time.Second, 10*time.Second).Should(BeTrue())
+
+		})
+
+		It("Should successfully start 2 VMs with persistent reservation on the same LUN", func() {
+			By("Create 2 VMs with the SCSI disk")
+			vmi := libvmi.NewFedora(
+				libvmi.WithPersistentVolumeClaimLun("lun0", pvc.Name, true),
+			)
+			vmi.Namespace = util.NamespaceTestDefault
```

Comment:
The main reason I added, it is because the function [CreatePVandPVCwithSCSIDisk](https://github.com/kubevirt/kubevirt/blob/f9566fe4f85bb592b09faefb80b1406395043027/tests/storage/reservation.go#L208) has the namespace and I wanted to be sure they are created on the same namespace